### DISCUSSION TITLE
Zeta components

### DISCUSSION THREAD

**Discussion by docs-of-all-trades** [fuma-nama/fumadocs#203] [Q&A] [answered]
Hello! I'm wondering if it's possible to use the [UI components](https://next-docs-zeta.vercel.app/docs/ui/components/accordion) on an existing next.js repo without creating a zeta app from the start.

**Comment by fuma-nama** [comment, marked as answer]
Theoretically, you can, but have to install `next-docs-zeta` as it relies on the headless components from zeta. 

If you want a full-powered UI components library, use shadcn/ui or Radix UI directly.
